CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Javis.jl - Changelog
 
+# PR changes
+- changed render method for mp4 to use ffmpeg directly inplace of VideoIO
+- TODO change test to use ffprobe for  video length and we can get rid of VideoIO completely
```

Comment:
Can you add this todo in the code and remove it from the changelog?

Reply:
done